app/src/i18n/locales/de/auth.json

```diff
@@ -47,12 +44,20 @@
   "your-email": "Ihre Email-Adresse",
   "check-email-reset-details": " existiert, werden Sie eine Email mit Instruktionen für das Zurücksetzen Ihres Passworts erhalten. Wenn diese nicht eintrifft, überprüfen Sie bitte Ihren Spam-Ordner.",
```

Comment:
The english version is:  
"check-email-reset-details": ", you will get an email with instructions on resetting your password. If it doesn't arrive, be sure to check your spam folder.",

The German version seems off starting with " existiert, ..."
Better would be ", Sie werden eine Email mit Anweisungen zum Zurücksetzen in Ihrem Postfach erhalten. Falls Sie keine Email erhalten, überprüfen Sie bitte Ihren Spam-Ordner"